store/layer_generators/main.go

```diff
@@ -10,11 +10,11 @@ import (
 	"go/format"
 	"go/parser"
 	"go/token"
-	"html/template"
 	"io/ioutil"
 	"os"
 	"path"
 	"strings"
+	"text/template"
```

Comment:
`html/template` doesn't support `-`

Reply:
Actually is `text/template` is the correct option, I guess I used `html/template` because `goimports` autocompleted it, but makes no sense at all to use `html/template` here.